packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/components/FlipbookControls.js

```diff
@@ -0,0 +1,171 @@
+    :focus {
+      outline: 2px solid ${tint(0.5, props.theme.global.colors.brand)};
+    }
+
+    :hover {
+      outline: 2px solid ${tint(0.5, props.theme.global.colors.brand)};
+    }
+  `}
+`
+
+export const StyledFrameList = styled.fieldset`
+  height: 44px;
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
+  border: none;
+`
+
+const StyledButton = styled(Button)`
+  & > div {
+    flex-direction: column;
+  }
+`
+
+const backgrounds = { dark: 'dark-3', light: 'neutral-6' }
+
+const FlipbookControls = ({
+  currentFrame = 0,
+  locations = [],
+  onFrameChange = () => true,
+  theme
+}) => {
+  const activeLabel = useRef()
+  const frameList = useRef()
+
+  const { t } = useTranslation('components')
+
+  const handleFrameChange = frameIndex => {
+    onFrameChange(frameIndex)
+  }
+
+  const handlePrevious = () => {
+    if (currentFrame > 0) {
+      handleFrameChange(currentFrame - 1)
+    }
+  }
+
+  const handleNext = () => {
+    if (currentFrame < locations.length) {
+      handleFrameChange(currentFrame + 1)
+    }
+  }
+
+  return (
+    <Box background={backgrounds}>
+      <Grid columns={['120px', 'flex']} pad='10px'>
+        {/** Play/Pause & Speed go here */}
+        <Box />
+
+        {/** Image Thumbnails */}
+        <Box direction='row' justify='between'>
+          <StyledButton
+            disabled={currentFrame === 0}
+            icon={<FormPrevious />}
+            label={t(
+              'SubjectViewer.MultiFrameViewer.FrameCarousel.previousFrameLabel'
+            )}
+            onClick={() => handlePrevious()}
+            style={{
+              border: 'none',
+              padding: 0
+            }}
+          />
+          <StyledFrameList ref={frameList}>
+            {locations?.length &&
+              locations.map((location, index) => {
+                const mimeType = Object.keys(location)[0]
+                const url = location[mimeType]
+                const activeFrame = currentFrame === index
+
+                return (
+                  <Box
+                    key={`${url}-${index}`}
+                    ref={activeFrame ? activeLabel : null}
+                    height='40px'
+                    width='40px'
+                    margin='0 5px'
+                    style={{
+                      position: 'relative'
+                    }}
+                  >
+                    <input
+                      id={`frame ${index}`}
+                      checked={activeFrame}
+                      name='frame'
+                      onChange={() => handleFrameChange(index)}
+                      type='radio'
+                      style={{
+                        position: 'absolute',
+                        top: 0,
+                        left: 0,
+                        margin: 0,
+                        height: '100%',
+                        width: '100%',
+                        opacity: '1%'
+                      }}
+                    />
```

Comment:
Using radio buttons to navigate slides was a bad idea that we've unfortunately copied across from PFE. I wonder if we should take this opportunity to get rid of this pattern? Leaving it in seems like it's just making more maintenance overhead for ourselves.

Radio buttons are confusing in the tutorial navigation, which is also used in the image-and-text viewer. They're probably confusing here too.
https://github.com/zooniverse/front-end-monorepo/pull/3533#pullrequestreview-1077041116

Reply:
What would you ideally replace radio buttons with in this case?

Reply:
The volunteer who found this pattern confusing was a blind volunteer, who came across the list of radio buttons after opening the tutorial, thought they were being asked to make a choice as part of the classification and became very confused.

In the tutorial, the radio buttons were also awkward because they're after the content that they control, rather than before it (https://github.com/zooniverse/front-end-monorepo/issues/1262.) That's also the case here. In the multi-frame viewer, with the controls on the left, you choose an image then move onto that image as you naturally tab through the page. So this pattern can be awkward for sighted keyboard users too.

Reply:
I agree buttons with aria-labels "Swtich to frame 1" or "Display frame 2" could work. I initially stuck with the radio buttons because only one frame should be selected at a time. Are radio buttons in a field set here confusing because the viewer is not a < form > ?

Reply:
I think part of the problem is that, visually, they aren't radio buttons. As a sighted user, how am I supposed to figure out that they are controlled with the cursor keys, and not with Tab and Enter/Space?

Reply:
The [WAI-ARIA Carousel design pattern](https://www.w3.org/WAI/ARIA/apg/patterns/carousel/) uses tabs for the picker, with the image as the tabpanel.

[Example tabbed carousel control](https://www.w3.org/WAI/ARIA/apg/example-index/carousel/carousel-2-tablist.html) (with code.)

Reply:
Thanks that carousel is a great example. I'll do some button refactoring, and then re-request review.